modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -400,11 +375,11 @@ When working with multiple currencies, be aware of which currency your sObject u
 
 === Limits on API Calls
 
-You need to know the rate limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
+You must know the rate limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
```

Comment:
```suggestion
You must know the parameters of the rate-limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
```

Reply:
Changing to "scope of."